sonata-project/form-extensions/1.4/config/packages/sonata_form.yaml

```diff
@@ -0,0 +1 @@
+../../../1.0/config/packages/sonata_form.yaml
```

Comment:
Links are generally confusing. 

I dont mind if you duplicate the 2 lines of config here.